scripts/ci-build-azure-ccm.sh

```diff
@@ -46,8 +46,8 @@ setup() {
 
     # the azure-cloud-provider repo expects IMAGE_REGISTRY.
     export IMAGE_REGISTRY=${REGISTRY}
-    pushd "${AZURE_CLOUD_PROVIDER_ROOT}" && TAG=$(git rev-parse --short=7 HEAD) &&
-      IMAGE_TAG_CCM="${IMAGE_TAG_CCM:-${TAG}}" && IMAGE_TAG_CNM="${IMAGE_TAG_CNM:-${TAG}}" &&
+    pushd "${AZURE_CLOUD_PROVIDER_ROOT}" && IMAGE_TAG=$(git rev-parse --short=7 HEAD) &&
```

Comment:
I wonder if we want to change this local `IMAGE_TAG` var name to something more semantic like `GIT_SHA`, to clarify that what we're doing here is:

1) We anticipate a specific image tag to be passed in for each component (via `IMAGE_TAG_CCM` and `IMAGE_TAG_CNM`)
2) If any one of the above vars aren't passed in, we fall back to use the image tag identified by the current checked out commit SHA

Thoughts?